### ISSUE TITLE
PT-BR translation is incorrect

### ISSUE THREAD

**Issue by luiztux** [preactjs/preact-www#1169]
Hey guys.
As the title indicates, the PT-BR translation is incorrect compared to the English version.

Here is the **pt-BR** version:

>Preact fornece a mais leve abstração de Virtual DOM possível em cima do DOM. A web é uma plataforma estável - **É tempo de pararmos de reimplementá-la em nome da segurança**.
Preact é também um cidadão de primeira-classe da plataforma web. Compara a Virtual DOM e o próprio DOM, registra event handlers de verdade, e funciona bem com outras bibliotecas.

![preact-ptbr-version](https://github.com/user-attachments/assets/06e75447-e69d-49f5-bf2e-fbec1dd965f6)

And here the **original** version:

>Preact provides the thinnest possible Virtual DOM abstraction on top of the DOM. It builds on stable platform features, registers real event handlers and plays nicely with other libraries.
Preact can be used directly in the browser without any transpilation steps.

![preact-original-version](https://github.com/user-attachments/assets/74a78579-40c9-4158-a745-cc4fe33f7b99)

If you put this into a translator, you will see that it is not correct.

Another thing that caught my attention is:
>**É tempo de pararmos de reimplementá-la em nome da segurança**

It's the same as writing in English:
>**It's time we stop reimplementing it in the name of security**

In other words, who wants to stop reimplementing something in the name of security when security is crucial? OMG 🤯

Therefore, the translation is placing preact as something insecure.

**Comment by rschristian** [comment]
If you speak Portuguese, we happily accept contributions!
